src/types/types.ts

```diff
@@ -510,6 +512,11 @@ export type Settings = {
   embedders?: Embedders;
   searchCutoffMs?: SearchCutoffMs;
   localizedAttributes?: LocalizedAttributes;
+
+  /** Enable facet searching on all the filters of an index. */
```

Comment:
Should we also add a comment to mention it's introduced with the v1.12, as you did on your other PR? 👀

Reply:
Great catch! I will add those comments 🔥